crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -338,25 +338,49 @@ def edit
   end
 
   def update
-    unless request.post?
-      raise ActionController::UnknownHttpMethod.new("POST is required to update proposal #{params[:id]}")
-    end
+    respond_to do |format|
+      format.json do
+        unless request.post?
```

Comment:
wouldn't it be better to have some nice answer here, instead of an exception on the server side, and a site not found in html?